testutils/channel_opts.go

```diff
@@ -193,6 +193,13 @@ func (o *ChannelOpts) SetRelayHosts(rh relay.Hosts) *ChannelOpts {
 	return o
 }
 
+// SetRelayLocal sets the channel's relay local handlers for service names
+// that should be handled by the relay channel itself.
+func (o *ChannelOpts) SetRelayLocal(relayLocal ...string) *ChannelOpts {
+	o.ChannelOptions.RelayLocalHandlers = relayLocal
```

Comment:
I think the `Set` prefix on the function should replace. If it was `AddRelayLocal` it would make more sense to append, but I don't think we need `AddRelayLocal` right now, since we set all the service names we'd like handled locally at once.